starter/script.js

```diff
@@ -1 +1,212 @@
-// put code here!
+//Mythical Beauty War
+
+//Goal is to get all the makeup, hair, and clothing neccessities for the Princess. And bring
+// them back so that she can go to THE red carpet event.
+
+//Items to get:
+//Elf workers gift = heels/shoes
+//Unicorn Rainbow throwup = lip sticks
+//Medusa hair = necklace
+//Dragon scales = dress
+//Fairy dust = shimmer and highlights
+//Mermaid hair = extentions or weave//Phoenix ashes = smokey eyes
+
+//First I need something to hold items: a dictionary/key:value pair.
+var backPack = {'Unicorn Rainbow Lipstick':0, 'Mermaid Weave':0, 'Dragon Scale Dress': 0,
+  'Medusa necklace':0, 'Stilletoes by ELF': 0, 'SmokeyEye by PHOENIX': 0, 'Fairy Face Shimmer': 0}
+
```

Comment:
What is your reasoning for using an object to store your inventory? Nothing wrong with the way you have it now, but it might be simpler to initialize `backPack` as an empty array and `push` items to it as the story progresses.